rust/src/section.rs

```diff
@@ -90,12 +90,12 @@ impl Section {
         SectionBuilder::new(name, range)
     }
 
-    pub fn name(&self) -> BnString {
-        unsafe { BnString::from_raw(BNSectionGetName(self.handle)) }
+    pub fn name(&self) -> String {
```

Comment:
If sections are named with the sections taken from the loaded binary, I don't think there's any guarantee they'll be UTF-8 names.

Doing a lossy conversion here will make it so that `view.section_by_name(section.name()))` fails. That doesn't seem great.